docs/library/processors/delays/allpassdelay/example-0.py

```diff
@@ -0,0 +1,20 @@
+from signalflow import *
+graph = AudioGraph()
+
+#-------------------------------------------------------------------------------
+# Using AllpassDelay to add a delay effect to a simple melodic sequence.
+# The original oscillator can be heard in the left channel.
+# The delay effect can be heard in the right channel.
+#-------------------------------------------------------------------------------
+clock = Impulse(1.0)
+sequence = Sequence([ 60, 62, 64, 65, 67, 69, 71, 72 ], clock)
+frequency = MidiNoteToFrequency(sequence)
+oscillator = TriangleOscillator(frequency)
+envelope= ASREnvelope(0, 0.2, 0.3, 1.0, clock)
```

Comment:
Couple of small things in this one:
 - `AllpassDelay`: The `max_delay_time` doesn't need to be specified as it defaults to 0.5
 - Missing space between `envelope=` (actually, I spotted the same missing space in a couple of the examples, I guess a copy/paste relic)

Reply:
Great spot, think I've fixed `envelope=` across the board